azure-mgmt-network/HISTORY.rst

```diff
@@ -3,12 +3,18 @@
 Release History
 ===============
 
-XXXXXXXXXXXX
-++++++++++++
+2.0.0rc2 (2018-04-03)
++++++++++++++++++++++
 
 **Features**
 
 - All clients now support Azure profiles.
+- API version 2018-02-01 is now the default
```

Comment:
Should this version be made as default for models as well?
https://github.com/Azure/azure-sdk-for-python/blob/53ded1052dc1ce3058194fc9980f0d3d2cd0b170/azure-mgmt-network/azure/mgmt/network/models.py#L7